packages/kbn-i18n/src/react/provider.tsx

```diff
@@ -50,29 +54,43 @@ function translateFormattedMessageUsingPseudoLocale(message: string) {
 /**
  * If pseudo locale is detected, default intl.formatMessage should be decorated
  * with the pseudo localization function.
- * @param child I18nProvider child component.
  */
-function wrapIntlFormatMessage(child: React.ReactNode) {
-  return React.createElement(
-    injectI18n(({ intl }) => {
+function wrapIntlFormatMessage() {
+  let isIntlOverridden = false;
```

Comment:
__issue:__ I'm starting to recover in memory why we didn't have `isIntlOverridden` before - intl can be re-created when we have several providers in the tree and hence we should override for every such provider. Currently that breaks pseudo-localization for Home/Tutorials pages.

Reply:
removed `isIntlOverridden`